Gopkg.toml

```diff
@@ -50,3 +50,7 @@
 [[override]]
   name = "github.com/json-iterator/go"
   version = "1.1.4"
+
+[[constraint]]
+  name = "github.com/kubernetes-csi/csi-lib-utils"
+  version = "0.2.0"
```

Comment:
Adding a constraint seems premature to me. This could still be added if (and only) if updating pulls in some incompatible version.

As it stands now, someone will have to remember to remove this constraint at some point or we'll forever be on a potentially old and insecure version.

Just my two cents, too late now anyway as this was already merged.